README.md

```diff
@@ -19,57 +19,93 @@ Eval JS-expression or require CommonJS modules and JSON files with ease.
 [david]:         https://david-dm.org/node-eval/node-evalenb/enb
 [david-img]:     https://img.shields.io/david/node-eval/node-eval/master.svg
 
-## Usage example:
+## Usage
 
-### Simple
+### JS-expression
 
 ```js
-var nodeEval = require('node-eval');
+const nodeEval = require('node-eval');
 
-console.log(nodeEval('42 * 42')); // 1764
+nodeEval('42 * 42'); // 1764
 ```
 
 ### CommonJS
-```js
-var safeEval = require('node-eval');
 
-var requireContent =
+```js
+const nodeEval = require('node-eval');
+const requireContents =
 `
-    var p = require('../package.json');
+    const package = require('./package.json');
+
     module.exports = {
-        name: p.name
+        name: package.name
     };
 `;
 
-var package = safeEval(requireContent);
-console.log(package.name); // 'node-eval'
+nodeEval(requireContents); // { name: 'node-eval' }
```

Comment:
What about path to file to provide required info to resolve relative paths inside evaluating code?

Reply:
Feels like we need one more example with resolving paths.